ingenialink/ethercat/network.py

```diff
@@ -9,7 +9,13 @@
 from threading import Thread
 
 import ingenialogger
-import pysoem  # type: ignore
+
+try:
+    import pysoem  # type: ignore
+except ImportError as e:
+    if isinstance(e, ModuleNotFoundError):
```

Comment:
Is this the only thing that should not be masked? I have no idea what errors can happen during an import